docs/DeveloperGuide.md

```diff
@@ -315,6 +315,122 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
+
+####Use case: Find a stock by name
```

Comment:
Should you add a space between `####` and `Use` so that the format would be consistent with the other files?

Reply:
I think it would be good to standardise.

Reply:
Thanks! Have done so in newest commit.